apps/core-app/src/renderer/src/views/omni-panel/ai-actions.ts

```
@@ -0,0 +1,219 @@
+  IntelligenceInvokeResult
+} from '@talex-touch/tuff-intelligence'
+import type {
+  OmniPanelContextSource,
+  OmniPanelDesktopContextCapsule
+} from '../../../../shared/events/omni-panel'
+import {
+  resolveDesktopContextInput,
+  summarizeDesktopContextCapsule
+} from '../../../../shared/intelligence/desktop-context-capsule'
+
+export type OmniPanelAiActionId =
```

ESLint

[error] '=' should be placed at the beginning of the line.

(style/operator-linebreak)